index.bs

```diff
@@ -4693,7 +4693,7 @@ partial interface MLGraphBuilder {
     1. Set |desc|.{{MLOperandDescriptor/dimensions}} to |outputShape|.
     1. *Make graph connections:*
         1. Let |output| be the result of [=creating an MLOperand=] given [=this=] and |desc|.
-        1. Let |operator| be an [=operator=] for the |op| pooling operation, given |options|.
+        1. Let |operator| be an [=operator=] for the |op| operation, given |options|.
```

Comment:
Intentional? "the op operation" sounds funny (kinda like saying PIN number or ATM machine).

Reply:
Intentional, for consistency - everything ends up being `for the "string" operation` once variables are expanded. There are worse examples e.g. "realm _realm_". I had to make my lint tool's check for duplicate words look at the source not final text.